deploy/ansible/roles/ethereum/tasks/main.yml

```diff
@@ -55,9 +80,8 @@
 
 
 - name: Deploy relay registry and master contract
-  shell:
-    _raw_params: >
-      ./gradlew runPreDeployEthereum --args='{{ addrs.stdout }}'
+  shell: "./gradlew runPreDeployEthereum --args='{{ addrs.stdout.split(' ') | select('match', '0x.*') | list | join(' ') }}'"
```

Comment:
I got address from previous commands without leading 0x

Reply:
maybe you can add task for copy or creation file with ethereum testnet passwords?